be/src/formats/utils.h

```diff
@@ -48,9 +48,19 @@ bool is_ready(std::future<R> const& f) {
 
 namespace formats {
 
+// file format
 inline static const std::string PARQUET = "parquet";
 inline static const std::string ORC = "orc";
 inline static const std::string CSV = "csv";
+inline static const std::string TEXTFILE = "textfile";
+
+// compression codec
+inline static const std::string COMPRESSION_CODEC = "compression_codec";
```

Comment:
We have enum in thrift and protobuf . String is not a good way

Reply:
updated